api/controllers/OpenEAppController.js

```diff
@@ -202,6 +206,16 @@ const OpenEAppController = {
             throw new Error('User not authorised to download this receipt');
         }
     },
+
+    _calculateRejectedDocs(casebookResponse) {
+        let rejectedDocs = 0;
+        for (const document of casebookResponse.documents) {
```

Comment:
could casebookResponse.documents be undefined here?  Probably needs a guard condition to check this first then return 0 if so

Reply:
It's not showing as outdated in GIthub but the code for this section has definitely changed:
https://github.com/UKForeignOffice/loi-application-service/pull/317/files#diff-9e383893e624afb9360415d5317a4a07c9360970f77ec0dc5ba0e9bef6e4c784R225

Reply:
Thanks, I can see the changes reflected in the actual commit: https://github.com/UKForeignOffice/loi-application-service/pull/317/commits/e9408622342e7a9c2df7acacfd78266c64852c40